subprojects/announce/announce.gradle.kts

```diff
@@ -21,4 +21,6 @@ dependencies {
     testRuntime(project(":plugins"))
 }
 
-useTestFixtures()
+testFixtures {
+    from(project = ":core")
```

Comment:
:x: The argument name doesn't add much value here. I would remove it for consistency.